### ISSUE TITLE
Idea: Checkbox for hiding the own name in 3rd person view

### ISSUE THREAD

**Issue by novemr** [cyberjunk/meridian59-dotnet#53]
I'd really like to hide my name in 3rd person view, it takes up way too much space

![image](https://cloud.githubusercontent.com/assets/12058341/7848240/65dfe740-04ca-11e5-9b9c-9bc93bd3d0e3.png)

**Comment by AlleyCat1976** [comment]
I agree here, on my screen my name is even much lower and basically touches the top of my head. Its a eye sore
